lib/web/conn_upgrade.go

```diff
@@ -21,22 +21,24 @@ package web
 import (
 	"context"
 	"io"
+	"log/slog"
 	"net"
 	"net/http"
 	"slices"
 	"sync"
 	"time"
 
+	"github.com/gobwas/ws"
```

Comment:
Do we have any plan to migrate all WS code to gobwas? Using multiple libraries to do the same thing sounds "not like a good thing".

Reply:
The original plan was to move to `gorilla/websocket` on the api client side after deprecating the legacy ALPN upgrade. But I was discussing with Edoardo that we may want to keep the legacy ALPN upgrade for performance reasons (planning to do some perf on this). Then this problem came.

"github.com/gobwas/ws" provides functionality to tune things at lower levels, potentially better performance too. However `gorilla/websocket` is more popular and we use it in many other places other than here. 

Since I will be doing some perf testing related to this, I can compare two lib. If `github.com/gobwas/ws` performance is on par or better, then we can replace `gorilla/websocket` before next major release testing.